code/online_challenges/src/project_euler/problem_004/problem_004.java

```diff
@@ -0,0 +1,30 @@
+    public class problem_004 {
+
+      public static void main( String[] args) {
+
+               int prod,temp,sum,rem,lar = 0,i,j;
+
+               for( i = 100; i <= 999; i++) {
+
+                  for( j = 100; j <= 999; j++) {
```

Comment:
Same as above.

Reply:
Thank you so much @mansiag for the suggestions. :smile:  I really appreciate it.